tests/runtime/for

```diff
@@ -92,6 +92,10 @@ NAME map with continue
 PROG begin { @map["foo"] = 1; @map["bar"] = 2; print("start"); for ($i : @map) { if ($i.0 == "bar") { continue; } print($i.0); }  }
 EXPECT_REGEX .*start\nfoo\n\n
 
+NAME map with return
```

Comment:
I think this test can be simplified with a range based for loop and just a single `begin` probe. Also you need an EXPECT_NONE for the print statement after the return in the loop.